### PR TITLE
Various Text improvements

### PR DESCRIPTION
**Pull request by Maascha**

### CODE REVIEW

File: src/pages/CreateCourse.tsx [hunk 1]
```diff
@@ -325,7 +325,7 @@ const CreateCourse: React.FC = () => {
     const _getCourseData = useCallback(
         () => ({
             description,
-            schooltype: studentData?.me?.student?.schooltype || 'other',
+            schooltype: studentData?.me?.student?.schooltype || 'Sonstige',
```
Comment (Jonasdoubleyou) [inline src/pages/CreateCourse.tsx:328]: 'Sonstige' is not a value of the schooltype enum (https://github.com/corona-school/backend/blob/dca404611ab5a85922b3c29605e5729b005a2854/common/entity/SchoolType.ts#L9). That you were able to set it to an invalid value suggests that the typing is wrong here :/

Comment (Jonasdoubleyou) [reply src/pages/CreateCourse.tsx:328]: If this is displayed wrongly, we should properly translate all the places the value appears in the UI.

File: src/types/lernfair/SchoolType.ts [hunk 2]
```diff
@@ -20,5 +20,5 @@ export const getSchoolTypeKey: (name: string) => string = (name) => {
             return schooltype.label;
         }
     }
-    return 'other';
+    return 'Sonstige';
```
Comment (Jonasdoubleyou) [inline src/types/lernfair/SchoolType.ts:23]: this is the label, not the key

File: src/modals/Tags.tsx [hunk 3]
```diff
@@ -67,7 +67,7 @@ const Tags: React.FC<Props> = ({ isOpen, onClose, selections, onSelectTag, onDel
                 </Modal.Body>
                 <Modal.Footer>
                     <Row space={space['1']}>
-                        <Button onPress={onClose}>{t('ok')}</Button>
+                        <Button onPress={onClose}>{t('back')}</Button>
```
[the remark this answers is not in the document]
Comment (Maascha) [reply src/modals/Tags.tsx:70]: Fair, good catch

File: src/lang/helperwizard/de.ts [hunk 4]
```diff
@@ -80,19 +80,19 @@ const helperwizard = {
     homeworkHelpStudent: {
         title: 'Hausaufgabenhilfe',
         content:
-            'Wir testen ein neues Angebot: <b>die Hausaufgabenhilfe</b>. Hier bekommen Schüler:innen einmalig, spontane, direkte Hilfe. Du findest, das klingt super? Dann freuen wir uns, wenn du mitmachst!<br>Komme dafür <b>im März von Montag-Donnerstag um 15:50</b> in die Vorbesprechung und unterstütze zwischen <b>16-17 Uhr</b> Schüler:innen in deinen Fächern für je ca. 15 Minuten bei ihren Hausaufgaben.',
+            'Bei der <b>Hausaufgabenhilfe</b> bekommen Schüler:innen einmalig, spontane, direkte Hilfe bei ihrer Frage zu den Hausaufgaben. Du findest, das klingt super? Dann freuen wir uns, wenn du mitmachst!<br>Komme dafür einfach <b>von Montag-Donnerstag um 15:50</b> in die Vorbesprechung und unterstütze zwischen <b>16-17 Uhr</b> Schüler:innen in deinen Fächern für je ca. 15 Minuten bei ihren Hausaufgaben. Schau vorbei, wir freuen uns auf dich!',
         buttons: ['Zur Hausaufgabenhilfe (Zoom-Registrierung)'],
```
[the remark this answers is not in the document]
Comment (Maascha) [reply src/lang/helperwizard/de.ts:84]: done

File: src/pages/pupil/single-course/PupilCourseButtons.tsx [hunk 5]
```diff
@@ -143,7 +143,7 @@ const PupilCourseButtons: React.FC<ActionButtonProps> = ({
                 <CourseConfirmationModal
                     headline={t('single.global.courseInfo')}
                     confirmButtonText={t('single.signIn.button')}
-                    description={t('single.signIn.description')}
+                    description={(<Trans i18nKey="single.signIn.description" components={{ b: <b />, br: <br /> }} />) as any}
```
Comment (Jonasdoubleyou) [inline src/pages/pupil/single-course/PupilCourseButtons.tsx:146]: That 'as any' cast is dangerous, better to change the type of the Modal's description property to ReactNode

**Review by Jonasdoubleyou — COMMENTED** [review]
(sorry, one small thing left I overlooked)

File: src/pages/pupil/single-course/PupilCourseButtons.tsx (hunk 5 above)
Comment (Maascha) [reply src/pages/pupil/single-course/PupilCourseButtons.tsx]: Done, thanks for catching!

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**